build_examples.cmake

```
@@ -0,0 +1,38 @@
+
+option(XMP_WITH_DEMO_PLAYER_SDL1      "Build also demo player for XMP (SDL1 needed)" OFF)
+if(XMP_WITH_DEMO_PLAYER_SDL1)
+    add_executable(xmpsdl
+        examples/player-sdl.c
+    )
+    add_dependencies(xmpsdl XMP_IF)
+
+    find_package(SDL REQUIRED)
+    if(TARGET SDL::SDL)
+        set(SDL_LIBRARY SDL::SDLmain SDL::SDL)
```

`SDL::SDLmain` doesn't seem to be provided by the standard `FindSDL` script.

```suggestion
        set(SDL_LIBRARY SDL::SDL)
```